### ISSUE TITLE
Installation Issue -- XSPEC version?

### ISSUE THREAD

**Issue by mikemoss3** [cositools/cosipy#173]
Hello! 

I am trying to install cosipy following docs. When I try `pip install cosipy` I run into an error (see output log file attached). 
[install-log.txt](https://github.com/cositools/cosipy/files/14885974/install-log.txt)

It shows that something goes wrong during the astromodels installation step and then starts to flag XSPEC versions. I recently updated my HEASOFT and XSPEC, so I have XSPEC version 12.13.1, which is not one of the listed "assumed" XSPEC versions in the warning message. I'm not sure if that has anything do to with it. 

I apologize in advance if this is an obvious fix I didn't see!

**Comment by israelmcmc** [comment]
Hi @mikemoss3. Cosipy doesn't require heasoft. It's only used by astromodels when you want to use some of the heasoft models, which we're not currently using for the data challenge. The easiest solution is to deactivate it before installing cosipy (iirc, I think you need to unset HEADAS). If you do need the heasoft models, then I think that the only solution is to install another version.

**Comment by mikemoss3** [comment]
Hi @israelmcmc. Following your advice I tried unsetting HEADAS and I believe that worked. 

However, I then had another installation issue since I was missing hdf5 library.
Specifically, I had to install the library via conda (i.e., `conda install h5py`) and the installation looks to have completed fine. 
Installing via pip (i.e., `pip install h5py`) didn't  work, but I think it would have worked if I also properly set HDF5_DIR environment variable to point to where the hdf5 header file would be. I'm not sure if it is mentioned anywhere that hdf5 is required for cosipy (or maybe everyone has it and I just didn't for some reason). 

(Also, sorry if this is the wrong thread to attach this to!)
